loafwallet.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@JasperHeist This should be reverted and check the .gitignore.  Let's talk on making sure your env matches mine.